content/concept-transaction-cost.md

```diff
@@ -69,9 +69,24 @@ The `rippled` server uses a variety of heuristics to estimate which transactions
 
 * Transactions must be properly-formed and [authorized](reference-transaction-format.html#authorizing-transactions) with valid signatures.
 * Transactions with an `AccountTxnID` field cannot be queued.
-* A single sending address can have at most 10 transactions queued at the same time. In order for a transaction to be queued, the sender must have enough XRP to pay all the XRP costs of all the sender's queued transactions including both the `Fee` fields and the sum of the XRP that each transaction could send. If a transaction affects how the address authorizes accounts, no other transactions from the same address can be queued behind it. _(New in [`rippled` 0.32.0](https://github.com/ripple/rippled/releases/tag/0.32.0))_
+* A single sending address can have at most 10 transactions queued at the same time. In order for a transaction to be queued, the sender must have enough XRP to pay all the XRP costs of all the sender's queued transactions including both the `Fee` fields and the sum of the XRP that each transaction could send. [New in: rippled 0.32.0][]
+* If a transaction affects how the sending address authorizes accounts, no other transactions from the same address can be queued behind it. [New in: rippled 0.32.0][]
 * If the transaction includes a `LastLedgerSequence` field, the value of that field must be at least **the current ledger index + 2**.
 
+#### Fee Averaging ####
+
+[New in: rippled 0.33.0][]
+
+If a sending address has one or more transactions queued, that sender can "push" the existing queued transactions into the open ledger by submitting a new transaction with a high enough transaction cost to pay for all of them. Specifically, the new transaction must increase the average transaction cost of the queued transactions from the same sending address, including itself, so that each of those transactions would be included in the open ledger if the `Fee` value of each of those transactions were equal to the average. The transactions must still follow the other [queuing restrictions](#queuing-restrictions) and the sending address must have enough XRP to pay the transaction costs of all the queued transactions.
```

Comment:
This is one of those cases where the math is relatively straightforward but hard to explain concisely, especially because it accounts for the fee escalating. I think it's a little clearer if we talk about "total" instead of "average".

`Specifically, the new transaction must increase the total transaction cost of the queued transactions from the same sending address, including itself, to cover the separate open ledger fee of each of those transactions as it gets added to the open ledger (accounting for the open ledger fee increasing as each transaction gets added).`

Meh.